### ISSUE TITLE
FEM: numbers disappear in color legend labels

### ISSUE THREAD

**Issue by jurpoc** [FreeCAD/FreeCAD#17719]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

My deepest congrats for the amazing work done to develop FreeCAD. I am running some non-linear FEM simulations and I have some problems with data representation.

FEM color legend labels disappear after saving a document.

Situation before saving a document.
![Screenshot from 2024-11-06 22-39-15](https://github.com/user-attachments/assets/74a68e02-5851-420d-9b54-06ff3be5dba6)

and after saving a document.
![Screenshot from 2024-11-06 22-42-07](https://github.com/user-attachments/assets/65ed0fb3-8c65-4989-bf22-0670a9fca756)

If the document is closed and opened again the legend labels reappear.

This happens for all simulations I run (Analysis type: Static).

### Full version info

```shell
OS: Ubuntu 24.04.1 LTS (ubuntu:GNOME/ubuntu/xcb)
Architecture: x86_64
Version: 1.1.0dev.39127 (Git) AppImage
Build type: Release
Branch: main
Hash: 1b1e355b9b8c22ae8ca01c383f6b57cdac8a7c04
Python 3.11.10, Qt 5.15.15, Coin 4.0.3, Vtk 9.3.0, OCC 7.8.1
Locale: English/United States (en_US) [ OS: English/United States (en_US) ]
Stylesheet/Theme/QtStyle: unset/FreeCAD Classic/Qt default
Installed mods: 
  * Render 2024.9.25
```

### Subproject(s) affected?

FEM

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
cc @FEA-eng

**Comment by jurpoc** [comment]
Yes, Finite Element Method (FEM) is more related to the maths behind. I should wrote Finite Element Analysis (FEA).

**Comment by luzpaz** [comment]
@FEA-eng should this be closed as a dupe ?

**Comment by FEA-eng** [comment]
> @FEA-eng should this be closed as a dupe ?

Yes, I think so.

**Comment by luzpaz** [comment]
Closed as a duplicate of #11665
